packages/webpack-cli/__tests__/info.test.js

```diff
@@ -0,0 +1,13 @@
+const { sync: spawnSync } = require('execa');
+const path = require('path');
+
+describe('Info', () => {
+    it('should run with cli', () => {
+        const { stdout, stderr } = spawnSync(path.resolve(__dirname, '../bin/cli.js'), [], {
+            cwd: path.resolve(__dirname),
+            reject: false,
+        });
+        expect(stdout).toBeTruthy();
+        expect(stderr).toBeFalsy();
```

Comment:
Can we tests that current output contains information from `info` pcakge, i.e. check what we have fields - `Binaries`/`Browsers`/`System`/`npmGlobalPackages`/`npmPackages`

Reply:
It's a integration test, info package as a whole is already tested.

Reply:
check `test/info` directory.

Reply:
Please improve tests, we can do regression(s) and `webpack info` can potential output any information (for example output version of package) and tests only on not empty output is not enough

Reply:
Okay I got it now, adding them my bad!